src/apps/mmfinance/cronos/MMfinance.pool.cache-manager.ts

```diff
@@ -0,0 +1,142 @@
+        if (code === '0x') return false;
+
+        const [symbol] = await Promise.all([
+          multicall
+            .wrap(lpTokenContract)
+            .symbol()
+            .catch(_err => ''),
+        ]);
+        // console.log(symbol, factoryAddressRaw);
+
+        // We've deprecated V1 support since the liquidities are low now (also our zap does not support V1)
+        // const V2_FACTORY_ADDRESS = '0xd590cC180601AEcD6eeADD9B7f2B7611519544f4';
+        // const isV2Pair = factoryAddressRaw.toLowerCase() === V2_FACTORY_ADDRESS;
+        const isCakeLp = symbol === 'MEERKAT-LP';
+        if (!isCakeLp) return null;
+        // console.log(lpTokenAddress);
+        return lpTokenAddress;
+      }),
+    );
+
+    return compact(allAddresses);
+  }
+
+  // @CacheOnInterval({
+  //   key: `apps-v3:${MMFINANCE_DEFINITION.id}:chef-v2-pool-addresses`,
+  //   timeout: 15 * 60 * 1000,
+  // })
+  // private async getChefV2PoolAddresses() {
+  //   const network = Network.CRONOS_MAINNET;
+  //   const chefAddress = '0x6bE34986Fdd1A91e4634eb6b9F8017439b7b5EDc';
+  //   const chefContract = this.contractFactory.MMfinanceChefV2({ address: chefAddress, network });
+
+  //   const provider = this.appToolkit.getNetworkProvider(network);
+  //   const multicall = this.appToolkit.getMulticall(network);
+  //   const numPools = await multicall.wrap(chefContract).poolLength();
+
+  //   const allAddresses = await Promise.all(
+  //     range(0, Number(numPools)).map(async v => {
+  //       const lpTokenAddressRaw = await multicall.wrap(chefContract).lpToken(v);
+  //       const lpTokenAddress = lpTokenAddressRaw.toLowerCase();
+  //       const lpTokenContract = this.contractFactory.MMfinancePair({ address: lpTokenAddress, network });
+
+  //       // Some EOAs exist on the MasterChef contract; calling these breaks multicall
+  //       const code = await provider.getCode(lpTokenAddress);
+  //       if (code === '0x') return false;
+
+  //       const [symbol, factoryAddressRaw] = await Promise.all([
+  //         multicall
+  //           .wrap(lpTokenContract)
+  //           .symbol()
+  //           .catch(_err => ''),
+  //         multicall
+  //           .wrap(lpTokenContract)
+  //           .factory()
+  //           .catch(_err => ''),
+  //       ]);
+
+  //       // We've deprecated V1 support since the liquidities are low now (also our zap does not support V1)
+  //       const V2_FACTORY_ADDRESS = '0xd590cC180601AEcD6eeADD9B7f2B7611519544f4';
+  //       const isV2Pair = factoryAddressRaw.toLowerCase() === V2_FACTORY_ADDRESS;
+  //       const isCakeLp = symbol === 'Cake-LP';
+  //       if (!isV2Pair || !isCakeLp) return null;
+
+  //       return lpTokenAddress;
+  //     }),
+  //   );
+
+  //   return compact(allAddresses);
+  // }
+
+  @CacheOnInterval({
+    key: `apps-v3:${MMFINANCE_DEFINITION.id}:static-pool-addresses`,
+    timeout: 15 * 60 * 1000,
+  })
+  private async getStaticPoolAddresses() {
+    return [''];
+  }
```

Comment:
You can remove this method and the commented out method. Easy to add back in later.